test/differentiable.lean

```diff
@@ -8,7 +8,7 @@ by simp
 example : differentiable ℝ (λ (x : ℝ), exp ((sin x)^2) - exp (exp (cos (x - 3)))) :=
 by simp
 
-example (x : ℝ) : deriv (λ (x : ℝ), (cos x)^2 + (sin x)^2) x = 0 :=
+example (x : ℝ) : deriv (λ (x : ℝ), (cos x)^2 + 1 + (sin x)^2) x = 0 :=
```

Comment:
Were these changes necessary? It seems like a regression.

Reply:
From the error message [in the failed build](https://github.com/leanprover-community/mathlib/runs/1528526191), my guess would be that the original test can be proved with `by simp`?

Reply:
With the new lemmas `simp` solved the goal by simplifying `(cos x)^2 + (sin x)^2` to `1`.